functional-tests/test-bed/src/main/java/org/arquillian/smart/testing/ftest/testbed/project/Project.java

```diff
@@ -36,6 +42,32 @@ public ProjectConfigurator configureSmartTesting() {
         return new ChangeApplier(repository).applyAsCommits(changeDescriptions);
     }
 
+    Project checkoutBranch(String branchName) {
+        try {
+            git.checkout().setName(branchName).call();
```

Comment:
There is no need to defend anything here (why it was done this way, where it was written to be done like this etc.). The conclusion about the code is what matters now. I think having it as a one-liner in the test would make it cleaner. WDYT?